docs/UserGuide.md

```diff
@@ -58,13 +58,13 @@ ___
 
 <div style="page-break-after: always;"></div>
 
-## GUI Breakdown
+## Graphical User Interface (GUI) Breakdown
 
 ![](images/UG-images/gui.JPG)
 
-1. **Menu Bar:** Here the user have the option to to change the application theme
+1. **Menu Bar:** Here the user has the option to change the application theme
 
-2. **Tabs Bar:** Here the user can change the tabs to view their anime list filtered based on their watch status. (watching, towatch, finished)
+2. **Tabs Bar:** Here the user can change tabs to view their animes filtered based on an anime's watch status (watch status can be one of `watching`, `to watch`, or `finished`)
```

Comment:
dont need the () as they are explained in the `Status` section already

Reply:
the () is originally there, i only added a space between to and watch. also at this point we have not yet introduced the term `watch status`. i think we should include it so as to avoid confusion bc it is for cs2101